### PR TITLE
Optimum v1.2: Support for Accelerated inference with Transformers Pipelines

### PR DESCRIPTION
**Pull request by philschmid**
# What does this PR do? 

This PR adds a supporting blog post for the Optimum v1.2 release, where we add the first support for an accelerated inference with transformers pipeline. 
This blog post is created to showcase a nice e2e example on how to optimize/quantize models and use them with transformers pipeline. 
But it also should increase SEO for accelerated inference, transformers, onnxruntime... To serve as a nice source of information. 

We would like to share & release this tomorrow 05/10. 

P.S. Did reviews with @jeffboudier and @lewtun on the content. 

Would love to get some additional feedback from you @BritneyMuller and @feconroses if you find time.

### CODE REVIEW

**Comment by BritneyMuller** [comment]
This looks great @philschmid! Nice job. 

The only suggestion I have is to perhaps leave out "amongst the [current limitations](https://www.notion.so/Accelerated-Inference-with-Optimum-https-github-com-huggingface-optimum-and-Transformers-Pipeli-1c8fc6770aca417aa04f02ae64fad132) are" on line 408. +Currently unable to access that Notion page.

**Comment by fxmarty** [comment]
This is cool stuff!

Thee link in 

> Check out [End-to-End Tutorial on accelerating RoBERTa for question-answering including quantization and optimization](https://www.notion.so/Accelerated-Inference-with-Optimum-https-github-com-huggingface-optimum-and-Transformers-Pipeli-1c8fc6770aca417aa04f02ae64fad132) for more details

is private for me, but maybe this is on purpose.

**Comment by fxmarty** [comment]
I think there is still the warning in case an `ORTModel` is passed to a transformers pipeline that it is an unsupported model ( https://github.com/huggingface/transformers/blob/main/src/transformers/pipelines/base.py#L863-L883 ), maybe better to wait until this is fixed as well.

File: optimum-inference.md [hunk 1]
```diff
@@ -0,0 +1,416 @@
...
+
+question = "What's my name?"
+context = "My name is Philipp and I live in Nuremberg."
+pred = onnx_qa(question, context)
+
+```
+
+In the first release, we added [support for Onnx Runtime](https://huggingface.co/docs/optimum/main/en/onnxruntime/modeling_ort) but there is more to come!
+These new `ORTModelForXX` can now be used with the [transformers pipelines](https://huggingface.co/docs/transformers/main/en/main_classes/pipelines#pipelines). They are also fully integrated into the [Hugging Face Hub](https://huggingface.co/models) to push and pull optimized checkpoints from the community. In addition to this, you can use the [ORTQuantizer](https://huggingface.co/docs/optimum/main/en/onnxruntime/quantization) and [ORTOptimizer](https://huggingface.co/docs/optimum/main/en/onnxruntime/optimization) to first quantize and optimize your model and then run inference on it.
+Check out [End-to-End Tutorial on accelerating RoBERTa for question-answering including quantization and optimization](https://www.notion.so/Accelerated-Inference-with-Optimum-https-github-com-huggingface-optimum-and-Transformers-Pipeli-1c8fc6770aca417aa04f02ae64fad132) for more details.
```
Comment (fxmarty) [inline optimum-inference.md:76]: This link is private to me, but maybe it is on purpose.

File: optimum-inference.md [hunk 2]
```diff
@@ -0,0 +1,416 @@
...
+
+```python
+from transformers import AutoTokenizer, pipeline
+from optimum.onnxruntime import ORTModelForQuestionAnswering
+
+tokenizer = AutoTokenizer.from_pretrained(onnx_path)
+model = ORTModelForQuestionAnswering.from_pretrained(onnx_path)
+
+onnx_qa = pipeline("question-answering", model=model, tokenizer=tokenizer)
+prediction = optimum_qa(question="What's my name?", context="My name is Philipp and I live in Nuremberg.")
```
Comment (echarlaix) [inline optimum-inference.md:241]: Typo : I guess it should be `onnx_qa` instead of `optimum_qa`

**Pull request merged**